va/utf8filter.go

```diff
@@ -0,0 +1,13 @@
+package va
+
+// replaceInvalidUTF8 replaces all invalid UTF-8 encodings with
+// Unicode REPLACEMENT CHARACTER.
+func replaceInvalidUTF8(input []byte) string {
+	var ret string
```

Comment:
its a little nit picky but I think a Go 1.10+ https://golang.org/pkg/strings/#Builder is generally a better choice for building up a string result

Reply:
+1 for doing this, it should (hopefully) not get used _that frequently_, but it's considered best practice to use `strings.Builder` so you don't have to keep assigning more memory and copying the previous string into it each time you extend the string. Just for the sake of the next time we need to do something like this it'd be good to have done this already.